packages/babel-traverse/src/path/conversion.ts

```diff
@@ -151,7 +158,7 @@ export function arrowFunctionToExpression(
     specCompliant?: boolean | void;
     noNewArrows?: boolean;
   } = {},
-) {
+): NodePath<Exclude<t.Function, t.Method | t.ArrowFunctionExpression>> {
```

Comment:
`path.arrowFunctionToExpression` is an API, should we defer the changes to minor?

Reply:
I think we have modified these internal plugin APIs in the past even in patch releases, and this change isn't adding "new functionality" but just exposing more info from an existing functionality.

However, if you prefer to wait that's fine